# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpcom/build/XPCOMInit.cpp
# Commit: 8bee937821e3
# Full Hash: 8bee937821e3725b922352a0493f53b5e431c3d0
# Author: Toshihito Kikuchi <tkikuchi@mozilla.com>
# Date: 2021-05-25 09:34:31
# Regressor Bug: 1586236
# File Overlap Count: 1
# Description:
#   Bug 1711610 - Should not trigger the memory pressure event if the available commit space is not low.  r=gsvelto
#   
#   After bug 1586236, we use the memory resource notification object to detect a low
#   memory situation on Windows, which is signaled when the available physical memory
#   is low.  If the available physical memory is low, however, it's possible that there
# ==============================================================================

diff -r e24c277bcb6c -r 8bee937821e3 xpcom/build/XPCOMInit.cpp
--- a/xpcom/build/XPCOMInit.cpp	Mon May 24 21:07:40 2021 +0000
+++ b/xpcom/build/XPCOMInit.cpp	Mon May 24 21:29:46 2021 +0000
@@ -85,7 +85,6 @@
 #include "base/message_loop.h"
 
 #include "mozilla/ipc/BrowserProcessSubThread.h"
-#include "mozilla/AvailableMemoryTracker.h"
 #include "mozilla/ClearOnShutdown.h"
 #include "mozilla/CountingAllocatorBase.h"
 #include "mozilla/UniquePtr.h"
@@ -462,7 +461,6 @@
   mozilla::SharedThreadPool::InitStatics();
 
   mozilla::scache::StartupCache::GetSingleton();
-  mozilla::AvailableMemoryTracker::Init();
 
   // Notify observers of xpcom autoregistration start
   NS_CreateServicesFromCategory(NS_XPCOM_STARTUP_CATEGORY, nullptr,
